pom.xml

```diff
@@ -17,6 +17,10 @@
             <url>http://opensource.org/licenses/MIT</url>
         </license>
     </licenses>
+    <properties>
+        <jenkins.version>1.580.1</jenkins.version>
+        <jenkins-test-harness.version>1.580.1</jenkins-test-harness.version>
```

Comment:
If I do that, I have to update the junit test classes to the new htmlunit.  It won't compile if I use 2.1.  I could make those changes here, but I was thinking it would be handled in a different pull request.